cmd/sedna-lc/app/server.go

```diff
@@ -86,17 +92,17 @@ func runServer() {
 		return
 	}
 
-	dm := manager.NewDatasetManager(c, Options)
+	dm := dataset.NewDatasetManager(c, Options)
 
-	mm := manager.NewModelManager(c)
+	mm := model.NewModelManager(c)
 
-	jm := manager.NewJointInferenceManager(c)
+	jm := jointinference.NewJointInferenceManager(c)
 
-	fm := manager.NewFederatedLearningManager(c)
+	fm := federatedlearning.NewFederatedLearningManager(c)
 
-	im := manager.NewIncrementalJobManager(c, dm, mm, Options)
+	im := incrementallearning.NewIncrementalJobManager(c, dm, mm, Options)
 
-	lm := manager.NewLifelongLearningJobManager(c, dm, mm, Options)
+	lm := lifelonglearning.NewLifelongLearningJobManager(c, dm, mm, Options)
```

Comment:
Can you abstract these mangers into same interface instead of different manager different arguments?